oojs.js

```diff
@@ -10,3 +10,23 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist{
+  constructor(title, song){
+    this.title = title;
+    this.song = song;
+    }
+  addSong(){
+  return `Your song was added to the Playlist ${this.song}`;
+  }
+}
+const music = new Plalist("Music");
+music.addSong();
+
+class Album{
+  constructor(artist, song){
+    super(artist, song);
```

Comment:
Here `super` is connected to the parent class and should be given parameters for it's constructor.
The `Playlist` constructor takes a `title` and a `song`, so passing `artist` here will make that the album title (`this.title`) the same as the artist name when it should have only been assigned to `this.artist`